packages/html-spec/src/loader/index.js

```diff
@@ -0,0 +1,3 @@
+const componentJson = require('./loader.json');
```

Comment:
```suggestion
// We should require and export loader-container as well.

const loaderJson = require('./loader.json');
const loaderContainerJson = require('./loader-container.json')

exports.loader = [ loaderJson, loaderConteinerJson ];
```

Reply:
Also we should comply with the JSON schema outlined in this PR https://github.com/telerik/kendo-themes/pull/4069